source/_posts/Planetscape-a-dystopian-escape-game-for-36C3.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Planetscape
+date: 2020/01/30
+---
+
+# Planetscape, a dystopian escape game for 36C3
+
+![][img:header]
+
+<!--
+## Summary
```

Comment:
Bit confused about this. Is hexo displaying this somehow?

Reply:
It's rendered in the HTML page but as a comment. It was the summary but I'm not sure it's needed. Would you keep it as summary or not?